crates/wit-parser/src/resolve.rs

```diff
@@ -1745,3 +1754,183 @@ impl<'a> MergeMap<'a> {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::Record;
+    use anyhow::Result;
+
+    use super::*;
+
+    const EMPTY_RESOLVE: &str =
+        r#"{ "worlds": [], "interfaces": [], "types": [], "packages": [] }"#;
+
+    const JSON_RESOLVE: &str = r#"{
+    "worlds": [
+        {
+        "name": "world1",
+        "docs": { "contents": null },
```

Comment:
Could `docs` be skipped if the contents are null?

Reply:
Sure. Is there any plans to add additional fields to the `Docs` struct? Could it be an optional string?

Reply:
Hm I don't think this would require a separate PR? I'm hoping to change this output in this PR by dropping the `"docs": ...` if it's empty. I think that can be done with `#[serde(skip_if = "...")]` or a similar field attribute name.

Reply:
Done.